canonical-websites/partners.ubuntu.com
# Sprint work so far

Done:
phone section renamed
Software programme: copy updates
new IoT page
renamed /partner-programmes -> /programmes

QA:
Check that the IA makes sense according to the new docs. Copy updates from copydocs should be included where applciable.
